moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -158,7 +158,6 @@ void Servo::setSmoothingPlugin()
     RCLCPP_ERROR(logger_, "Smoothing plugin could not be initialized");
     std::exit(EXIT_FAILURE);
   }
-  resetSmoothing(getCurrentRobotState());
```

Comment:
The positions from `getCurrentRobotState()` were all-zeros since this call occurred early and we didn't wait for a valid robot state.